<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentAdd_Dialog</class>
 <widget class="QDialog" name="SegmentAdd_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>209</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>216</width>
    <height>209</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>216</width>
    <height>209</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Segment Adding</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
	background-color: rgb(43, 43, 43);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>32</number>
        </property>
        <property name="verticalSpacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="SegID_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Segment ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="SegIDInput">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="PointSID_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Point I IDï¼š</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="PointSID_Input">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="PointEID_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Point J ID:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="PointEID_Input">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="Thickness_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Thickness:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="Thk_Input">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="MatID_Label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Mat. ID:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="MatID_Input">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SegmentAdd_pushButton">
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SegAddCancel_pushButton">
          <property name="styleSheet">
           <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}
</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SegIDInput</tabstop>
  <tabstop>MatID_Input</tabstop>
  <tabstop>PointSID_Input</tabstop>
  <tabstop>PointEID_Input</tabstop>
  <tabstop>Thk_Input</tabstop>
  <tabstop>SegmentAdd_pushButton</tabstop>
  <tabstop>SegAddCancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
